validation-test/stdlib/CUUID.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-run-stdlib-swift
+// REQUIRES: executable_test
+
+// REQUIRES: OS=linux-gnu
+
+import Swift
+import StdlibUnittest
+
+
+import CUUID
```

Comment:
Could you add a comment to the test that explains what it does?  "Make sure that the module map for the uuid.h header works."